environment/container/kubernetes/kubernetes.go

```diff
@@ -50,9 +50,6 @@ func (c kubernetesRuntime) Running() bool {
 func (c kubernetesRuntime) runtime() string {
 	return c.guest.Get(environment.ContainerRuntimeKey)
 }
-func (c kubernetesRuntime) kubernetesVersion() string {
```

Comment:
This is actually temporary and it will soon be used again. Can we temporarily disable linting for this?